Sources/RealDeviceMap/Modell/Account.swift

```diff
@@ -438,6 +443,26 @@ class Account: WebHookEvent {
             cooldownSQL = ""
         }
 
+        Account.lockoutLock.lock()
+        let now = Date()
+        Account.lockouts.removeAll { (lockout) -> Bool in
+            lockout.untill <= now
+        }
+        let locked = Account.lockouts.filter { (lockout) -> Bool in
+            return lockout.device != device
+        }.map { (lockout) -> String in
+            return lockout.account
+        }
+        Account.lockoutLock.unlock()
```

Comment:
Here we could potentially leave it locked for the entire duration to completely avoid collisions.
Like this it is still possible to get collisions in rare cases when accounts pull at the exact same time.
Locking until the db responds would slow down getting of new accounts on large maps.